scripts/irods/test/test_irepl.py

```diff
@@ -13,13 +13,16 @@
 from .. import test
 from ..configuration import IrodsConfig
 
-class Test_IRepl(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass')]), unittest.TestCase):
+class Test_IRepl_Special(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass')]), unittest.TestCase):
```

Comment:
better/different name than 'special'?

Reply:
me too - but not sure how much of that is Python convention vs unittest module requirements vs Zoey defaults persisting.

Reply:
Changed the name